packages/manager/core/app/src/api/nutanix.ts

```diff
@@ -76,4 +90,79 @@ export async function fetchNutanixMetaInfos(
   return { region: data.datacenter };
 }
 
+export async function getNutanix(serviceName: string): Promise<Nutanix> {
+  const response = await fetch(`/engine/api/nutanix/${serviceName}`);
+  return response.json();
+}
+
+export async function getServiceInfos(
```

Comment:
Can it be replaced with `/services` ?

Reply:
The use /serviceInfos is to get the `serviceId` with given `serviceName`. I need it to get this information and then call /services.